cache/blobs.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/containerd/containerd/diff/walking"
 	"github.com/containerd/containerd/leases"
 	"github.com/containerd/containerd/mount"
+	nydusify "github.com/containerd/nydus-snapshotter/pkg/converter"
```

Comment:
Why is this imported by default? IIUC this isn't used when `-tags=nydus` isn't specified, right?